src/client.c

```diff
@@ -1739,6 +1753,7 @@ int main(int argc, char *argv[]) {
     AppData *app_data = g_slice_new0 (AppData);
     app_data->rsh_cmd = "ssh";
     app_data->restraint_path = "restraintd";
+    app_data->restraint_port = 0;
```

Comment:
Initialization is not required since app_data was allocated and initialized using g_slice_mem0()

Reply:
I set the value to be explicit about the default for the argument. If I don't, one my wonder if I forgot, or if using the zeroed memory set by g_slice_new0 is OK.

Reply:
If it brings you comfort, I'm good.